content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -144,9 +144,432 @@ dependencies {
+
+```shell
+export OTEL_RESOURCE_ATTRIBUTES="key1=value1,key2=value2"
+```
+
+The service name is determined by the following precedence, in accordance with the OpenTelemetry
+[specification](https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/#otel_service_name):
+
+1. `otel.service.name` spring property or `OTEL_SERVICE_NAME` environment variable (highest
+   precedence)
+2. `service.name` in `otel.resource.attributes` system/spring property or `OTEL_RESOURCE_ATTRIBUTES`
+   environment variable
+3. `service.name` in `otel.springboot.resource.attributes` system/spring property
+4. `spring.application.name` spring property
+5. the default value `unknown_service:java` (lowest precedence)
+
+##### Exporter Properties
+
+| Feature         | Property                      | Default Value                        |
+|-----------------|-------------------------------|--------------------------------------|
+| Otlp Exporter   | otel.exporter.otlp.endpoint   | `localhost:4317`                     |
+|                 | otel.exporter.otlp.protocol   | `grpc`                               |
+|                 | otel.exporter.otlp.headers    |                                      |
+|                 | otel.exporter.otlp.timeout    | `1s`                                 |
+| Jaeger Exporter | otel.exporter.jaeger.endpoint | `localhost:14250`                    |
+|                 | otel.exporter.jaeger.timeout  | `1s`                                 |
+| Zipkin Exporter | otel.exporter.jaeger.endpoint | `http://localhost:9411/api/v2/spans` |
+
+The `otel.exporter.otlp.headers` property can be specified as a comma-separated list,
+which is compliant with the
+[specification](https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/#otel_exporter_otlp_headers).
+Similar to the resource attributes, the headers can be specified in `application.properties` or
+`application.yaml`:
+
+```yaml
+otel:
+  exporter:
+    otlp:
+      headers:
+        - key: "header1"
+          value: "value1"
+        - key: "header2"
+          value: "value2"
+```
+
+### Tracer Properties
+
+| Feature | Property                        | Default Value |
+| ------- | ------------------------------- | ------------- |
+| Tracer  | otel.traces.sampler.probability | `1.0`         |
+
+
+## Spring Support
+
+Auto-configuration is natively supported by Springboot applications. To enable these features in "vanilla" use `@EnableOpenTelemetry` to complete a component scan of this package.
+
+### Usage
+
+```java
+import io.opentelemetry.instrumentation.spring.autoconfigure.EnableOpenTelemetry;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableOpenTelemetry
+public class OpenTelemetryConfig {}
+```
+
+## Additional instrumentations
+
+### Manual Instrumentation Support - @WithSpan
```

Comment:
Perhaps rename this section _Instrumentation annotations_ and move it after the _Logging Instrumentation_ part?